docs/operator_convert.md

```diff
@@ -159,6 +160,50 @@ Options with an asterisk are required. Note that `enumerate` can be used by itse
  }
 ]
 ```
+## Automatically convert all "numeric" strings ignoring suffixed property
+
+### Rule
+
+```javascript
+{
+ source: 'digitalData.cart.price[(userId, available,basePrice,cartTotal,priceWithTax)]',
+ operators: [
+   { name: 'convert', enumerate: true, ignoreSuffixed: true }, 
```

Comment:
I guess we could omit `ignoreSuffixed` now that ignoring suffixes is the default.

Reply:
> LGTM. Left a couple small things to consider, but nothing required to merge this change.

Made those changes.